crates/uv/src/commands/publish.rs

```diff
@@ -109,3 +115,16 @@ pub(crate) async fn publish(
 
     Ok(ExitStatus::Success)
 }
+
+fn prompt_username_and_password() -> Result<(Option<String>, Option<String>)> {
+    let term = Term::stderr();
+    if !term.is_term() {
+        anyhow::bail!("Not a terminal");
```

Comment:
We should not error in this case, but return `(None, None)` and later in the publish code error if we don't have any credentials. uv often runs in non-interactive environments such as CI.

Reply:
Done